### PR TITLE
Docs for set_permissions, set_ownership, current_user helpers for postflight and uninstall_preflight blocks

### PR DESCRIPTION
**Pull request by vitorgalvao**
Refs #12977.

Note: do not forget to update examples in `doc/CASK_LANGUAGE_REFERENCE.md` after casks are updated.

### CODE REVIEW

**Comment by jawshooah** [comment]
These helpers are now available to both the `postflight` and `uninstall_preflight` blocks, as each includes the methods of `DSL::Staged`. The docs should probably reflect that.

**Comment by vitorgalvao** [comment]
Yes, I noticed that but wasn’t yet sure on how to go about doing it in a clear way. Calling it `Postflight and Uninstall_preflight Stanza Details` and `Postflight and Uninstall_preflight Mini-DSL` is a bit verbose and weird. Might be the best option for now, though.

**Comment by jawshooah** [comment]
The `suppress_move_to_applications` method actually *is* defined on `DSL::Postflight`, not `DSL::Staged`, so it's available in `postflight` blocks but not `uninstall_preflight`.

Maybe we should just have separate sections for `postflight` and `uninstall_preflight`, and swallow the duplication?

**Comment by vitorgalvao** [comment]
What about this solution? At least the `availability` section in the table should be enough to clarify everything. Just not sure `*flight` would be the clearest way to describe it, although it certainly is short.

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Cask Domain-Specific Language (DSL) which are not needed in most cases.
  * [Depends_on Stanza Details](#depends_on-stanza-details)
  * [Conflicts_with Stanza Details](#conflicts_with-stanza-details)
  * [Uninstall Stanza Details](#uninstall-stanza-details)
- * [Postflight Stanza Details](#postflight-stanza-details)
+ * [\*flight Stanza Details](#flight-stanza-details)
```
Comment (jawshooah) [inline doc/CASK_LANGUAGE_REFERENCE.md]: Link should point to `#flight-stanzas-details`

File: doc/cask_language_deltas.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ features which are available for the current Cask.
  * [`installer :script`](CASK_LANGUAGE_REFERENCE.md#installer-script)
  * [`license`](CASK_LANGUAGE_REFERENCE.md#license-stanza-details)
  * [`name`](CASK_LANGUAGE_REFERENCE.md#name-stanza-details)
- * [`postflight plist_set`](CASK_LANGUAGE_REFERENCE.md#postflight-stanza-details)
+ * [`*flight plist_set`](CASK_LANGUAGE_REFERENCE.md#flight-stanza-details)
```
Comment (jawshooah) [inline doc/cask_language_deltas.md]: Same as above; should be `#flight-stanzas-details`

File: doc/cask_language_deltas.md [hunk 3]
```diff
@@ -65,6 +65,10 @@ features which are available for the current Cask.
    * *stub* - currently just a synonym for `uninstall :delete`
  * [`zap`](CASK_LANGUAGE_REFERENCE.md#zap-stanza-details)
 
+### (1.1)
+ * [`*flight set_ownership`](CASK_LANGUAGE_REFERENCE.md#flight-stanza-details)
```
Comment (jawshooah) [inline doc/cask_language_deltas.md]: And again here and below.

File: doc/CASK_LANGUAGE_REFERENCE.md (hunk 1 above)
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: Ah, right, thank you. Adding the “s” was one of the last things, and forgot about the links.

**Comment by jawshooah** [comment]
Looks fine to me, apart from the minor fixes I pointed out above.

**Comment by vitorgalvao** [comment]
Merged as 55c31018069bdb4f6d984a65bb8e7b7a2e94e870.

**Pull request closed without merging**